featurebyte/routes/use_case/controller.py

```diff
@@ -66,3 +83,56 @@ async def delete_use_case(self, document_id: ObjectId) -> None:
             UseCase id to be deleted
         """
         await self.service.delete_document(document_id=document_id)
+
+    async def get_info(self, use_case_id: ObjectId) -> UseCaseInfo:
+        """
+        Get detailed information about a UseCase
+
+        Parameters
+        ----------
+        use_case_id: ObjectId
+            UseCase ID
+
+        Returns
+        -------
+        UseCaseInfo
+        """
+        use_case = await self.service.get_document(document_id=use_case_id)
+        context = await self.context_service.get_document(document_id=use_case.context_id)
+        target = await self.target_service.get_document(document_id=use_case.target_id)
+
+        author = None
+        if use_case.user_id and hasattr(self.user_service, "get_document"):
```

Comment:
do you think it might be easier to add a no-op `get_document` method into the user service here so that we don't have to do the `hasattr` check?

Reply:
good idea